layouts/_default/baseof.html

```diff
@@ -82,6 +82,37 @@ <h1 class="page-title">{{ .Title }}</h1>
     <script src="{{ $.Site.Params.enterprise_script }}"></script>
     {{ end }}
 
+    {{ if $.Site.Params.headings_enabled }}
```

Comment:
I would suggest calling the param `enableSubLinks` instead of  `headings_enabled`.